src/main/java/nextstep/session/domain/EnrolledStudents.java

```diff
@@ -1,19 +1,41 @@
 package nextstep.session.domain;
 
+import nextstep.payments.domain.PaymentPolicy;
 import nextstep.users.domain.NsUser;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class EnrolledStudents {
 
-    List<NsUser> students = new ArrayList<>();
+    private Long sessionId;
 
-    public void add(NsUser user) {
-        students.add(user);
+    public EnrolledStudents() {
+    }
+
+    public EnrolledStudents(Long sessionId, List<Long> students) {
+        this.sessionId = sessionId;
+        this.students = students;
+    }
+
+    List<Long> students = new ArrayList<>();
```

Comment:
```suggestion
    private List<Long> students = new ArrayList<>();
```

인스턴스 변수의 위치에 대한 관계 위반
sessionId 바로 아래 구현할 것을 추천